codebuild/bin/install_default_dependencies.sh

```diff
@@ -104,7 +104,7 @@ if [[ "$TESTS" == "integration" || "$TESTS" == "integrationv2" || "$TESTS" == "A
     if [[ ! -x "$OQS_OPENSSL_1_1_1_INSTALL_DIR/bin/openssl" ]]; then
       # Download and Install OQS OpenSSL for integration tests
       mkdir -p "$OQS_OPENSSL_1_1_1_INSTALL_DIR" ||true
-      codebuild/bin/install_oqs_openssl_1_1_1.sh "$(mktemp -d)" "$OQS_OPENSSL_1_1_1_INSTALL_DIR" "$OS_NAME" > /dev/null ;
+      codebuild/bin/install_oqs_openssl_1_1_1.sh "$(mktemp -d)" "$OQS_OPENSSL_1_1_1_INSTALL_DIR" "$OS_NAME" | head -50
```

Comment:
Doesn't this print the whole log except for the last 50 lines? Why do we want to do that?

Reply:
First 50 lines just to make sure it's running; I was having intermittent failures of this script with no output, it's impossible to know what's going on.

Reply:
Ah got it okay. First 50 lines makes more sense!